### PR TITLE
feat(presentational): add Tabs component

### PR DESCRIPTION
**Pull request by agory**

### CODE REVIEW

**Comment by mfrachet** [comment]
WIP so ?

**Comment by agory** [comment]
Normally, this pr can be reviewed. :smile: I added you as a reviewer.

File: src/web/components/Tabs/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import List from '../List';
+import Item from '../Item';
+import sizable from '../../hoc/Size';
+/* eslint-disable */
+// use of tabs component bulma with a whithout href
+
+class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.renderItem = (item, index) => (
```
Comment (mfrachet) [inline src/web/components/Tabs/index.js]: This is not the place to declare that function 😄

File: src/web/components/Tabs/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <Item className={this.props.index == index ? 'is-active' : ''}>
+        <a onClick={() => this.props.onSelected(index)}>{item}</a>
```
Comment (mfrachet) [inline src/web/components/Tabs/index.js]: No anonymous function as callback 😈

File: src/web/components/Tabs/Tabs.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import List from '../List';
+import Touchable from '../Touchable';
+
+describe('Tabs', () => {
+  let wrapper;
+  let onSelected;
+  let data;
+
+  beforeEach(() => {
+    data = ['Hello', 'Bonjour', 'Ciao'];
```
Comment (mfrachet) [inline src/web/components/Tabs/Tabs.test.js]: What do you think of something a bit powerful ?

```javascript
const tabs = [{
 key: 'Hello',
 component: () => <SomeComponent />
}]
```

Comment (mfrachet) [reply src/web/components/Tabs/Tabs.test.js]: So with the internal state management, `Tabs` is independant but respect its own responsability: display a content tab when its focused

**Comment by agory** [comment]
I made it statefull. Is my implementation correct?

File: stories/tabs.stories.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  { key: 'Hello', component: () => <Text>Hello</Text> },
+  { key: 'Bonjour', component: () => <Text>Bonjour</Text> },
+  { key: 'Ciao', component: () => <Text>Ciao</Text> },
+];
+
+storiesOf('Tabs', module)
+  .add('Basic Tabs', () => <Tabs data={data} onChange={action('Tabs')} />)
+  .add('with size', () => [
+    <Tabs data={data} size="small" onChange={action('small')} />,
+    <Tabs data={data} onChange={action('default')} />,
```
Comment (mfrachet) [inline stories/tabs.stories.js]: When we select a tab, a part of the screen should display the `component: ()`, it's not the case in the storybook

Comment (agory) [reply stories/tabs.stories.js]: My bad when you say :
```
const tabs = [{
 key: 'Hello',
 component: () => <SomeComponent />
}]
``` 
I thought the component field would be the content of the onglet.
I'm going to update it.

**Pull request merged**